engine/jni/Android.mk

```diff
@@ -35,4 +35,6 @@ LOCAL_LDLIBS            +=  -L$(SYSROOT)/usr/lib \
 
 LOCAL_CPP_EXTENSION     := .cxx .cpp .cc
 
+LOCAL_DISABLE_FATAL_LINKER_WARNINGS=true
+
```

Comment:
Besides, the file name of ```org_probedroid_Instrument.cc``` objects to match the automatically generated JNI header ```org_probedroid_Instrument.h```.  It would be better to rename it in ```Android.mk```. (Though the NDK is not case sensitive.)